workflow-dev/tpl/deis-builder-deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
             - name: BUILDER_STORAGE
               value: "{{ or (env "STORAGE_TYPE") (.storage)}}"
             - name: "DEIS_REGISTRY_LOCATION"
-              value: "{{ .registry_location }}"
+              value: "{{ or (env "REGISTRY_LOCATION") (.registry_location) }}"
```

Comment:
why not the familiar pattern of `{{ (env "REGISTRY_LOCATION") | default .registry_location }}`?

same with the below